File: CS/Dashboard_CreateGeoPointMap/Form1.cs
using System.Windows.Forms;
using DevExpress.DashboardCommon;
using DevExpress.DataAccess.ConnectionParameters;
using DevExpress.DataAccess.Sql;

namespace Dashboard_CreateGeoPointMap {
    public partial class Form1 : Form {
        public Form1() {
            InitializeComponent();

            Dashboard dashboard = new Dashboard();
            DashboardSqlDataSource dataSource = new DashboardSqlDataSource();
            CustomSqlQuery sqlQuery = new CustomSqlQuery("Cities", "select * from Cities");
            dataSource.ConnectionParameters = 
                new Access97ConnectionParameters(@"..\..\Data\Cities.mdb", "", "");
            dataSource.Queries.Add(sqlQuery);

            GeoPointMapDashboardItem geopointMap = new GeoPointMapDashboardItem();
            geopointMap.DataSource = dataSource;
            geopointMap.DataMember = "Cities";

            geopointMap.Area = ShapefileArea.WorldCountries;

            geopointMap.Latitude = new Dimension("Latitude");
            geopointMap.Longitude = new Dimension("Longitude");
            geopointMap.Value = new Measure("Population");

            dashboard.Items.Add(geopointMap);
            dashboardViewer1.Dashboard = dashboard;
        }
    }
}

File: VB/Dashboard_CreateGeoPointMap/Form1.vb
Imports System.Windows.Forms
Imports DevExpress.DashboardCommon
Imports DevExpress.DataAccess.ConnectionParameters

Namespace Dashboard_CreateGeoPointMap

    Public Partial Class Form1
        Inherits Form

        Public Sub New()
            InitializeComponent()
            ' Creates a new dashboard and a data source for this dashboard.
            Dim dashboard As Dashboard = New Dashboard()
            Dim dataSource As DataSource = New DataSource(New SqlDataProvider("Connection 1", New Access97ConnectionParameters("..\..\Data\Cities.mdb", "", ""), "select * from Cities"))
            ' Creates a Geo Point Map dashboard item and specifies its data source.
            Dim geopointMap As GeoPointMapDashboardItem = New GeoPointMapDashboardItem()
            geopointMap.DataSource = dataSource
            ' Loads the map of the world.
            geopointMap.Area = ShapefileArea.WorldCountries
            ' Provides city coordinates and corresponding populations.
            geopointMap.Latitude = New Dimension("Latitude")
            geopointMap.Longitude = New Dimension("Longitude")
            geopointMap.Value = New Measure("Population")
            ' Adds the Geo Point Map dashboard item to the dashboard and opens this
            ' dashboard in the Dashboard Viewer.
            dashboard.Items.Add(geopointMap)
            dashboardViewer1.Dashboard = dashboard
        End Sub
    End Class
End Namespace
